src/streamlink/plugins/youtube.py

```diff
@@ -203,14 +207,29 @@ def _create_adaptive_streams(self, info, streams, protected):
         return streams, protected
 
     def _find_channel_video(self):
-        res = http.get(self.url)
-        match = _channelid_re.search(res.text)
-        if not match:
-            return
+        res = http.get(self.url, headers={"User-Agent": useragents.CHROME})
+
+        datam = _ytdata_re.search(res.text)
+        if datam:
+            data = parse_json(datam.group(1))
+            # find the videoRenderer object, where there is a LVE NOW badge
+            for x in search_dict(data, 'videoRenderer'):
+                for blabel in search_dict(x.get("badges", {}), "label"):
+                    if blabel == "LIVE NOW":
```

Comment:
> if blabel == "LIVE NOW":

**won't work** as this string changes with every different youtube **language** version

maybe you can use the

> 'style': 'BADGE_STYLE_TYPE_LIVE_NOW'

maybe this work,
tested on
- https://www.youtube.com/user/france24
- https://www.youtube.com/user/Euronews

---

some urls from the header will work too